### PR TITLE
2.0 release of streams

### PR DESCRIPTION
**Pull request by ktoso**

### CODE REVIEW

File: news/_posts/2015-12-21-akka-streams-2.0-released.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+   32    7954    3082 Mathias
+   27    3932    4035 Viktor Klang
+   26   10634    3554 Endre Sándor Varga
+   26   10587   12318 Roland Kuhn
+   21     608     521 2beaucoup
+   12    3232     941 Johan Andrén
+   10    1635     884 Martynas Mickevičius
+    6     573      82 lolski
+    3      16       1 hepin
+    3      44      46 Martynas Mickevicius
```
Comment (drewhk) [inline news/_posts/2015-12-21-akka-streams-2.0-released.md]: add to the number above

File: news/_posts/2015-12-21-akka-streams-2.0-released.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+    2      35       3 Anton Karamanov
+    2     205       7 Gilad Hoch
+    2       2       2 netcomm
+    2     148      24 Catalin Ursachi
+    1       1       1 Johan Andrén
+    1       3       3 Fehmi Can Saglam
+    1      37       8 Frank Murphy
+    1     180     223 André Rüdiger
+    1       3      24 drewhk
```
Comment (drewhk) [inline news/_posts/2015-12-21-akka-streams-2.0-released.md]: that's me too

**Comment by drewhk** [comment]
LGTM

**Comment by ktoso** [comment]
Right, I'll sum up the numbers :)
Thanks for having a look!

File: news/_posts/2015-12-21-akka-streams-2.0-released.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+
+**Closing Remarks**
+
+We would like to give our special thanks and kudos to Alexander Golubev, who has continuously contributed a steady stream of new features to Akka Streams while we were busy working on the internals. Thanks! We also thank @2beaucoup for his continued effort in reviews and fixes in Http. And of course we thank all of the 42 individuals that contributed to this part of Akka’s success!
+
+The reference documentation for the 2.0 release can be found here for [Java](http://doc.akka.io/docs/akka-stream-and-http-experimental/2.0/java.html) and [Scala](http://doc.akka.io/docs/akka-stream-and-http-experimental/2.0/scala.html).
+
+The full list of changes since the last milestone is available under the [streams-2.0](https://github.com/akka/akka/issues?q=is%3Aissue+milestone%3Astreams-2.0+is%3Aclosed) and [http-2.0](https://github.com/akka/akka/issues?q=is%3Aissue+milestone%3Ahttp-2.0+is%3Aclosed) milestones on github for your reference.
+
+We would like to thank everyone contributed to this release of Akka Streams & HTTP since its 1.0 release a few months ago:
```
Comment (rkuhn) [inline news/_posts/2015-12-21-akka-streams-2.0-released.md]: everyone _who_ contributed

**Comment by ktoso** [comment]
Summed up and fixed typo.

release the kraken

**Pull request merged**